config.json

```diff
@@ -1,28 +1 @@
-{
-    "templates": {
-        "visuals": "templates/visuals",
-        "pbiviz": "templates/pbiviz.json.template",
-        "plugin": "templates/plugin.ts.template",
-        "package": "templates/package.json.template"
-    },
-    "generate": {
-        "apiVersion": "1.12.0"
-    },
-    "build": {
-        "precompileFolder": ".tmp/precompile",
-        "dropFolder": ".tmp/drop",
-        "js": "tmp.visual.js",
-        "css": "visual.css"
-    },
-    "package": {
-        "dropFolder": "dist"
-    },
-    "server": {
-        "root": "webRoot",
-        "assetsRoute": "/assets",
-        "privateKey":"certs/PowerBICustomVisualTest_private.key",
-        "certificate":"certs/PowerBICustomVisualTest_public.crt",
-        "pfx":"certs/PowerBICustomVisualTest_public.pfx",
-        "port":"8080"
-    }
-}
\ No newline at end of file
+{"templates":{"visuals":"templates/visuals","pbiviz":"templates/pbiviz.json.template","plugin":"templates/plugin.ts.template","package":"templates/package.json.template"},"generate":{"apiVersion":"1.12.0"},"build":{"precompileFolder":".tmp/precompile","dropFolder":".tmp/drop","js":"tmp.visual.js","css":"visual.css"},"package":{"dropFolder":"dist"},"server":{"root":"webRoot","assetsRoute":"/assets","privateKey":"certs/PowerBICustomVisualTest_private.key","certificate":"certs/PowerBICustomVisualTest_public.crt","pfx":"certs/PowerBICustomVisualTest_public.pfx","port":"8080","passphrase":"8595900307629099"}}
```

Comment:
Do we need to keep this file? Is it auto generated?

Reply:
We need to keep it. It isn't auto-generated. But tools puts passphrase into config.